pnpm-lock.yaml

```diff
@@ -46,10 +45,10 @@ importers:
         version: 5.69.0(react@18.3.1)
       '@uiw/codemirror-themes':
         specifier: ^4.23.10
-        version: 4.23.10(@codemirror/language@6.11.0)(@codemirror/state@6.5.2)(@codemirror/view@6.36.5)
+        version: 4.23.10(@codemirror/view@6.36.5)
       '@uiw/react-codemirror':
         specifier: ^4.23.10
-        version: 4.23.10(@babel/runtime@7.27.0)(@codemirror/autocomplete@6.18.6)(@codemirror/language@6.11.0)(@codemirror/lint@6.4.2)(@codemirror/search@6.5.3)(@codemirror/state@6.5.2)(@codemirror/theme-one-dark@6.1.2)(@codemirror/view@6.36.5)(codemirror@6.0.1)(react-dom@18.3.1(react@18.3.1))(react@18.3.1)
+        version: 4.23.10(@codemirror/autocomplete@6.18.6)(@codemirror/view@6.36.5)(react-dom@18.3.1(react@18.3.1))(react@18.3.1)
```

Comment:
@ryanclark What's the easiest way to test `e/web/teleport/src/AccessMonitoring/QueryEditor/QueryEditor.tsx` locally? It'd be nice to have a story, but for now I could get by with a patch that enables me to open it in my cluster.

I'm trying to turn off autoinstall of missing peer deps by default because of a misbehaving electron-builder dep (see https://github.com/gravitational/teleport/pull/53610#discussion_r2029115822). This causes some changes in the lockfile related to `@codemirror` packages, so I just wanted to confirm that the page is still working. `pnpm i --resolution-only` doesn't show any warnings about missing `@codemirror` peers, so I think it should be fine.

Reply:
You could probably just grab the `<CodeMirror />` component from there and move it somewhere you can test it, grabbing the `config` and the value state get/setter

That way it won't try to load a bunch of access monitoring stuff

Reply:
It renders, no errors are thrown and it shows autocomplete for SQL keywords, so I'm going to assume that it works. 👍

<img width="1280" alt="query-editor" src="https://github.com/user-attachments/assets/a7c26ab4-7e13-4235-8f9a-ab3e28fda786" />